app/controllers/application_controller.rb

```diff
@@ -37,12 +42,21 @@ def not_found
       raise ActionController::RoutingError.new('Not Found')
     end
 
-    def flash_success(message)
+    def flash_success(message = 'Your changes have been saved')
       flash[:success] = message
     end
 
     def flash_errors(record)
       flash[:error] = record.errors.full_messages.join("\n")
     end
 
+    # soon, we should consolidate ALL dashboards?
+    def current_dashboard_path
+      if current_user.platform_admin?
```

Comment:
This is bad.  It should include everyone manager and above, i.e:
`if current_user.roles.empty?`
and I think there was an issue with that... PRs welcome